data2bids.m

```diff
@@ -1542,7 +1542,7 @@
     begsample                   = table2array(events_tsv(:,{'begsample'}));
     endsample                   = table2array(events_tsv(:,{'endsample'}));
     onset                       = (begsample-1)./hdr.Fs;
-    duration                    = (endsample-begsample+1)./hdr.Fs; 
+    duration                    = (endsample-begsample)./hdr.Fs; 
```

Comment:
Can you please revert this? If an event lasts from sample N to sample N+1, I would say that its duration is 2 samples. So if it lasts from N to N, its duration is 1 sample.

Reply:
Okay, done